web/handler.py

```diff
@@ -190,12 +199,15 @@ def get_name(module_path: str) -> str:
         elif self.path == "/run-ai":
             if Handler.server_phase != "Configuration" or not exists("prev.data"):
                 self.send_redirect("/")
-            Handler.server_phase = "Starting"
-            with open("prev.data") as file:
-                config: dict[str, Any] = loads(file.read())
-            Handler.result_config.update(config)
-            Handler.config_lock.release()
-            self.send_redirect("/")
+            try:
+                Handler.server_phase = "Starting"
```

Comment:
Why not only the “with open” is in the try-except block, but also the Handler code?